example/test.js

```diff
@@ -0,0 +1,10 @@
+var css = [
+  '.ninja {',
+  '  visibility: hidden;',
+  '  color: black;',
+  '}'
+].join('\n')
+
+var css2jss = require('../lib/cssToJss.js')
```

Comment:
ok, i can update it, just i think, especially when using browserify, requiring just the `cssToJss` saves when it comes to file size, but it's probably not a lot anyway.